pocs/tests/test_observatory.py

```diff
@@ -37,6 +42,19 @@ def observatory(config, simulator, images_dir):
     return obs
 
 
+def test_camera_already_exists(observatory, config):
```

Comment:
I see you added this already. We try to keep PRs focused on a single thing and adding tests for the camera doesn't really have anything to do with the scheduler. Ok to leave for now but in the future I'd probably ask you to more to a separate PR.